chromatic/.eslintrc.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Do we need this? There is already a root eslint config.

Reply:
I removed it, bus since i did the saving of certain files take a long time. I remember a similar problem being solved by having this in a lower level folder. Or was that something else?

Reply:
If you define one here that extends the root level one, then that speeds up `eslint`.